src/rgw/rgw_rest_swift.cc

```diff
@@ -802,12 +802,12 @@ static int get_swift_account_settings(req_state * const s,
   const char * const acl_attr = s->info.env->get("HTTP_X_ACCOUNT_ACCESS_CONTROL");
   if (acl_attr) {
     RGWAccessControlPolicy_SWIFTAcct swift_acct_policy(s->cct);
-    int r = swift_acct_policy.create(store,
+    bool r = swift_acct_policy.create(store,
```

Comment:
A really, really minor thing: we could make this `const` but I'm still OK with the patch.